openrpc.json

```diff
@@ -843,6 +843,20 @@
           "$ref": "#/components/schemas/bytes32"
         }
       }
+    },
+    {
+      "name": "web3_clientVersion",
+      "description": "Returns the current client version.",
+      "summary": "Current client version.",
```

Comment:
Should we add a bit more here about what we mean by the client version in this case?